tasks/src/app/ordered-list/ordered-list.component.css

```
@@ -0,0 +1,3 @@
+.highlightingName {
```

```suggestion
.highlighting-name {
```
for css use kebab-case